directory_sso_api_client/user.py

```diff
@@ -0,0 +1,14 @@
+from directory_sso_api_client.base import BaseAPIClient
+
+
+class UserAPIClient(BaseAPIClient):
+
+    endpoints = {
+        'session_user': '/session-user/'
+    }
+
+    def get_session_user(self, session_id):
+        return self.get(
+            url=self.endpoints['session_user'],
+            headers={'USER_SESSION_ID': session_id}
```

Comment:
Django will do cookie based auth for us - we can provide session_id as a cookie with no extra cost. why send it as a header?